api/src/main/java/org/openmrs/module/sync2/api/model/configuration/WhitelistConfiguration.java

```diff
@@ -0,0 +1,51 @@
+package org.openmrs.module.sync2.api.model.configuration;
+
+import java.util.List;
+import java.util.Objects;
+
+public class WhitelistConfiguration {
+
+    private boolean enabled;
+    private List<String> instanceIds;
+
+    public WhitelistConfiguration() {}
```

Comment:
What do you think about Initialization the instanceIds variable with empty ArrayList?

Reply:
OK